new_platforms/include/oeenclave.h

```diff
@@ -2,39 +2,41 @@
 /* Licensed under the MIT License. */
 #pragma once
 #ifndef _OE_ENCLAVE_H
-# error include <openenclave/enclave.h> instead of including oeenclave.h directly
+#error include <openenclave/enclave.h> instead of including oeenclave.h directly
 #endif
 
 #ifndef OE_SIMULATE_OPTEE
 #include <stdio.h>
 #endif
-#include <stddef.h>
-#include <stdarg.h>
 #include <assert.h>
+#include <stdarg.h>
+#include <stddef.h>
 #include <stdint.h>
 
 #ifdef OE_SIMULATE_OPTEE
-# include <sal.h>
-# include "optee/enclave/Simulator/oeenclave.h"
+#include <sal.h>
+#include "optee/enclave/Simulator/oeenclave.h"
 #endif
 
-#include "tcps.h"
 #include <openenclave/bits/result.h>
+#include "tcps.h"
 
 typedef int64_t __int64_t;
 typedef uint64_t __uint64_t;
 typedef uint32_t __uint32_t;
 #include <sys/types.h>
 
-#ifndef __in_ecount
+#ifdef LINUX
```

Comment:
I think this change is wrong, as SAL vs no-SAL should not be about LINUX vs not-Linux.
It's more likely about whether compiling with VS or not.   So more likely
#ifndef _MSC_VER
would be more correct.

Reply:
Changed as suggested.